client/driver/docker.go

```diff
@@ -119,13 +107,10 @@ func NewDockerDriverConfig(task *structs.Task) (*DockerDriverConfig, error) {
 	if err := mapstructure.WeakDecode(task.Config, &driverConfig); err != nil {
 		return nil, err
 	}
-	if strings.Contains(driverConfig.ImageName, "https://") {
```

Comment:
I didn't understand this. This code converts `image = "https://example.com.redis:latest"` to `image = "example.com.redis:latest"`. But what situation users specify image name with `https://` like that?